Platform/Sophgo/SG2042Pkg/Library/PeiCoreInfoHobLib/PeiCoreInfoHobLib.inf

```diff
@@ -0,0 +1,44 @@
+## @file
+#  Library instance to create core information HOB

```

Comment:
I am not sure whether you really this and RiscVSmbiosDxe. type 44 specific to RISC-V is not ratified / frozen table (though referenced from smbios spec) and the current info what it provides already provided by other means now. So, see if you can avoid these modules by using standard SMBIOS module.

Reply:
Thank you for your suggestion! We will try to avoid these modules by using standard SMBIOS modules.